discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -18,24 +18,27 @@
 class Cluster:
 
     def __init__(self, api_client, cluster_name=None, additional_ntp_source=None,
-                 openshift_version="4.6", cluster_id=None):
+                 openshift_version="4.6", cluster_id=None, user_managed_networking=False, high_availability_mode="Full"):
```

Comment:
lets have high_availability_mode as class in consts with 2 values full and none

Reply:
OK

Reply:
OK